drop-in/src/main/java/com/adyen/checkout/dropin/DropInServiceResult.kt

```diff
@@ -81,6 +81,8 @@ sealed class DropInServiceResult : BaseDropInServiceResult() {
         override val reason: String? = null,
         override val dismissDropIn: Boolean = false
     ) : DropInServiceResult(), DropInServiceResultError
+
+    class ToPaymentMethodsList : DropInServiceResult()
```

Comment:
Wondering if we should add the payment methods list as an optional parameter here, to allow reloading drop-in with new payment methods

Reply:
I will check if I can get that to work